lib/components/chat_history/twitch/poll_event.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+import 'dart:math' as math;
+
+List<Widget> getPollsWidget(List<dynamic> choices, List<Widget> children,
+    bool isCompleted, TextStyle baseStyle) {
+  num totalVotes = 0;
+  num totalChannelPointsVotes = 0;
+  num totalBitVotes = 0;
+  num maxVotes = 0;
+
+  for (final entry in choices) {
```

Comment:
Can this logic be moved to the model? Same as in @juaoose's PR, if we move the computation stuff to the model it'll make it easier to test.